wordpress-mobile/WordPress-Android
# Add the UI for filtering posts by author

This PR adds the ability to filter the Posts List by author with the two options shown to the user being `Me` and `Everyone`. Fixes #5884

To test:
1. Go to the post list of a site
1. Next to the `TabLayout` you can tap the icon to change whether the list is showing all posts or just your own posts.
2. When you change the author filter you should see the posts shown change (if applicable)
